packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -90,21 +93,26 @@ class VsCode {
   //   $HOME/.vscode/extensions
   //   $HOME/.vscode-insiders/extensions
   static List<VsCode> _installedMacOS() {
-    final Map<String, String> stable = <String, String>{
-      fs.path.join('/Applications', 'Visual Studio Code.app', 'Contents'):
-          '.vscode',
-      fs.path.join(homeDirPath, 'Applications', 'Visual Studio Code.app',
-          'Contents'): '.vscode'
-    };
-    final Map<String, String> insiders = <String, String>{
-      fs.path.join(
-              '/Applications', 'Visual Studio Code - Insiders.app', 'Contents'):
-          '.vscode-insiders',
-      fs.path.join(homeDirPath, 'Applications',
-          'Visual Studio Code - Insiders.app', 'Contents'): '.vscode-insiders'
-    };
-
-    return _findInstalled(stable, insiders);
+    return _findInstalled(<_VsCodeInstallLocation>[
+      new _VsCodeInstallLocation(
+        fs.path.join('/Applications', 'Visual Studio Code.app', 'Contents'),
+        '.vscode'
```

Comment:
Minor nit: trailing commas here will mean that when we're finally able to run `dartfmt` on this repo, it'll keep this formatting.

Reply:
👍